spec/factories/quotes.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :quote do
+    author   Faker::Name.first_name
```

Comment:
You'll want to make these attributes lazy evaluated by using a block, or else every quote will have the name author , body and location. This likely doesn't matter if there is no uniqueness constraint but it may be confusing or unintended. 

e.g.
`author { Faker::Name.first_name }`